pkg/vtpm/src/server.cpp

```diff
@@ -150,24 +462,33 @@ int main(int argc, char *argv[])
         if ((sock = accept(listenSock, (struct sockaddr *)&clientAddr,
```

Comment:
Note that this will need to change before we ever allow two domUs on an edge node, because one can prevent the other from using the vTPM by just connecting to the port and never sending anything and never closing the connection.

One could potentially have a timeout in the reading from the socket, but it might make more sense to spawn one worker for every connection. To be discussed.